pkg/util/azureclient/authz/remotepdp/types.go

```diff
@@ -32,8 +32,14 @@ type SubjectAttributes struct {
 	AltSecId         string   `json:"altsecid,omitempty"`
 	IdentityProvider string   `json:"idp,omitempty"`
 	Issuer           string   `json:"iss,omitempty"`
+	ClaimName        string   `json:"_claim_names,omitempty"`
 }
 
+// GroupExpansion is the value to be used with ClaimName in SubjectAttributes
+// This value gives CheckAccess a hint that it needs to retrieve all the groups the principal belongs to
+// and then give the response based on all group entitlements.
+const GroupExpansion = "{\"groups\":\"ENGORGIO~!\"}" // ref: https://en.wikibooks.org/wiki/Muggles%27_Guide_to_Harry_Potter/Magic/Engorgio
```

Comment:
is it magic word documented in checkaccess api doc?

Reply:
Nope, the value can be anything.  The document has just "src1".

Reply:
src1 is what AAD would return when groups claim need expansion. I think we should stay true to this and add a comment that this value doesn't matter.

Reply:
Also, please include the ref to their doc